sovrin_client/test/anon_creds/test_public_repo.py

```diff
@@ -85,10 +90,11 @@ def submittedPublicRevocationKey(submittedPublicKeys):
 def testSubmitSchema(submittedSchemaDefGvt, schemaDefGvt):
     assert submittedSchemaDefGvt
     assert submittedSchemaDefGvt.seqId
-    # initial schema didn't have seqNo
-    submittedSchemaDefGvt = submittedSchemaDefGvt._replace(seqId=None)
-    assert submittedSchemaDefGvt == schemaDefGvt
-
+    # initial schema has stub seqno - excluding seqno from comparison
+    def withNoSeqId(schema):
+        schema._replace(seqId=None)
```

Comment:
You are missing a return before `schema._replace(seqId=None)` and so the next checks is equivalent to `assert None == None`